pogobuf/pogobuf.client.js

```diff
@@ -37,7 +39,7 @@ function Client() {
      * @param {string} authType - Authentication provider type (ptc or google)
      * @param {string} authToken - Authentication token received from authentication provider
      */
-    this.setAuthInfo = function(authType, authToken) {
+    this.setAuthInfo = function (authType, authToken) {
```

Comment:
There are a lot of these whitespace changes. Looks like an editor's auto-refactoring. As it's a style choice, and no other changes to these lines have been made, I'd suggest leaving the whitespace as it is. It just clutters up the merge.